samples/ambient-argo/README.md

```diff
@@ -0,0 +1,99 @@
+# Ambient Reference Architecture w/ Argo
+
+This repo contains a reference architecture for operating Istio Ambient Mesh with ArgoCD using GitOps.  It demonstrates best practices for leveraging Istio as part of an application platform.
+
+## :boom: DISCLAIMER
+
+Istio Ambient Mesh is still in Alpha, and is not suitable for production use.  Likewise, this reference architecture is of Alpha quality, and includes several rough edges, including:
+ * Cluster-Scoped upgrades cause known traffic loss, and have wide blast radius.
+ * The tag chart is forked from the primary istio repo, and needs to be merged and published
+ * CRDs are not currently upgraded
+
+## Getting Started
+
+This reference architecture assumes that you have an ArgoCD installation with:
+ * A [connected cluster](https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_cluster/) named `ambient-cluster`
+ * A connection to your repository (for private repos)
+
+To deploy Istio, supporting software, and the bookinfo sample application, copy this folder to the root of your repo and run:
+
+```bash
+read -p 'Please enter the URL to your repo:'
+OLD_REPO='{repo-placeholder}'
+find . \( -type d -name .git -prune \) -o -type f -name '*.yaml' -print0 | xargs -0 sed -i s,$OLD_REPO,$NEW_REPO,g
+argocd create application -f meta-application.json
+```
+
+## Repository Layout
+
+The meta-application.yaml file is an App-of-Apps that references all other applications needed for running Istio and the demo application via ArgoCD.  The diagram below demonstrates the deployment mechanism for each part of the platform.
+
+![architecture diagram][layout]
+
+## Principles
+
+The [GitOps Principles](https://opengitops.dev/) guide this reference architecture.  With the exception of [./meta-application.yaml](./meta-application.yaml) (the bootstrap file), Git is the source of truth for all components of our application.  Changes to all components of Istio, including the data plane, are initiated with a pull request.  Likewise, rollbacks are as simple as a revert.
```

Comment:
Out of scope for this specific PR but it'd be interesting to include auto-rollback with Argo given some user-specified application metrics or control plane metrics

Reply:
Yes!  Defining the SLA is the hard part, but I think we should totally look into this, and not just for argo